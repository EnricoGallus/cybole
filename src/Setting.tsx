import {useEffect} from "react";

function Setting() {
    useEffect(() => {
        const storedPreference = localStorage.getItem('darkModePreference');
    })

    return (
        <div>
            Setting
        </div>
    )
}

export default Setting;