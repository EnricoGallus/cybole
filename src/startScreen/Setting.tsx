import React from 'react';
import { Link } from 'react-router-dom';
import {Button} from "primereact/button";

const Setting = () => (
    <div>
        <div className="card">
            <h5>Settings</h5>
            <Link to="/">
                <Button label="Back to Project Selection" className="p-button-secondary" />
            </Link>
        </div>


    </div>
);

export default Setting;
