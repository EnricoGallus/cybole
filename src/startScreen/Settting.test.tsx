import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Setting from './Setting';

describe('Setting', () => {
    test('renders component', () => {
        const { container } = render(<Setting />, {
            wrapper: MemoryRouter,
        });

        expect(container).toMatchSnapshot();
    });
});
