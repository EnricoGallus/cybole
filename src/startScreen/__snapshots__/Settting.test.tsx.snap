// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Setting renders component 1`] = `
<div>
  <div>
    Setting
    <a
      class="btn btn-primary"
      href="/"
    >
      Back to Project Selection
    </a>
  </div>
</div>
`;
